pkg/cli/cmd/radinit/recipe.go

```diff
@@ -192,8 +188,3 @@ func getPortableResourceType(resourceType string) string {
 		return ""
 	}
 }
```

Comment:
can we get rid of getNormalizedResourceTypeFromPath() and getActualResourceType() functions and instead make availableDevRecipes() return list of string in the format of  Applications.<rp>/<resourcetype> e.g Applications.Datastores/mongoDatabases ?